incubator/hnc/pkg/validators/config.go

```diff
@@ -0,0 +1,182 @@
+package validators
```

Comment:
Should this filename be hncconfig? Otherwise it's ambiguous with hierarchyconfig. Ditto for all the names below.

Reply:
Good idea! Changed to `hncconfig` along with the struct name.